department-of-veterans-affairs/caseflow
# Connect Reader to v2 eFolder

connects #4166 
connects #4167 

Steps to validate:
1. Enable v2 for a user
FeatureFlag.enable!(:efolder_api_v2, users: ["css_id1"])
2. Go to the Reader's list documents page and ensure it is pulling documents from efolder v2 api
3. Ensure it works as expected
